packages/material/src/layouts/MaterialArrayLayout.tsx

```diff
@@ -40,114 +56,266 @@ import Avatar from '@material-ui/core/Avatar';
 import Paper from '@material-ui/core/Paper';
 import DeleteIcon from '@material-ui/icons/Delete';
 import ValidationIcon from '../complex/ValidationIcon';
+import { connect, Dispatch } from 'react-redux';
+import { AnyAction } from 'redux';
 
-export const MaterialArrayLayout =
-  ({
-     data,
-     path,
-     label,
-     schema,
-     createDefaultValue,
-     uischema,
-     errors,
-     addItem,
-     findUISchema,
-     removeItems,
-     renderers
-   }: ArrayControlProps) => {
+const paperStyle = { padding: 10 };
+const iconStyle: any = { float: 'right' };
+export class MaterialArrayLayout extends React.Component<
+  ArrayLayoutProps,
+  { expanded: string | boolean }
```

Comment:
You can have multiple expanded panels, but according to the type signature it seems as if we can have only one.

Reply:
now you can't ;-) i suggest to open a new issue for this if we want to support multiple open panels at once